media-sound/alsa-tools/Manifest

```diff
@@ -1,3 +1,9 @@
+AUX alsa-tools-1.0.29-output_tram_line-static.patch 377 SHA256 3fce2936bab1db37a80e0e88bf3aedb8c5ccfa7c581ff2a455e24ad1a3d518e5 SHA512 42283f6ef67df86693bb321ff27ed9301d998be395ff583ba3e496061cf72f0be3e19bc93c7ea6ec722c239ac8578c52dcc15028bc7fbe020241f9aaba396bf2 WHIRLPOOL 26e0f38756c17e27bd48dc5ef039a7011c1599bd5692f744fd7a0373fb0559b06589c50d47f8ffe33e80b19c1d501c246bfc1b378236758b0f11dd7771bedd6a
+AUX envy24control-config-dir.patch 593 SHA256 6cc8c284109839102a12a11de0e834865aa860460d3e387d71a9ae013ba8e1f9 SHA512 267be12fb2d9d561cc2746317beaf5168362005cb98a55d3efe5741dcd974b088bfa049f2ed185919083fe2e24596eec67de2f2984da5369b9aa0eac983b5cef WHIRLPOOL abcc73750027f595b70fa9471334d5bc3b0f326e7847d6fd12ef0ad1107435af8f3236ad5e9b4ab89a9091cd97e1c87258f635137959d3136c3e7780e12394eb
 DIST alsa-tools-1.0.28.tar.bz2 1675680 SHA256 76e59711c6d0f39cbddce83ce1ed8da00bad112fee021f94fa990d8685cc3761 SHA512 b5150edb5f5124d5ec256b7c3348207da1bb9847ab2f10297171d6a43f559a31429ef96608a2c670db75d269dd940ebf2b6e3cbfafde530de25cbd95dce31af2 WHIRLPOOL 20f8acf8edf3fa82f19e4098c520c155a87966272d769f29a6208b25fc80a831a6cfdb3c748406ced643b5de7fa4cfb43f3a62d634fd90512369b1f57ea9cb92
 DIST alsa-tools-1.0.29.tar.bz2 1715439 SHA256 94abf0ab5a73f0710c70d4fb3dc1003af5bae2d2ed721d59d245b41ad0f2fbd1 SHA512 c236e105106e65b9f02e5baed536fa33f8efcff21a75470127e2edacf0bbb3ec555131e146d9dbfc8da307355b372e7c2acaf8edaa98dc7880abd484fbdb2813 WHIRLPOOL 19d34f425ad45d540a0ccd2ae2c77bae810ee6039250f57296c280156c6e6ed8d3788f306975f6b06b8b3f377e714d644aed08c177bd9d83acf352ccb939a47d
 DIST alsa-tools-1.1.0.tar.bz2 1715113 SHA256 7d34558c590a50294b36576d257316a1ac5cd951eb8cd7d330e09f8cc757ab51 SHA512 caabbc9a8b494de6bba0dea6440b335b35e5c7dab4942ac36f127a75c2212ff98e7e38566444666edcf40c6c95e6d9797c421014fb5c5c292bb398780e6c9f87 WHIRLPOOL daf731da17c172275b484da37962d85569d70066392bb65f6b462a5f9f17784d204deec203d2466cbc08cb691689b7fc8cd664b370b39e233f1e8afb43c9d0ff
+EBUILD alsa-tools-1.0.28.ebuild 3501 SHA256 ba88efe7a1107b198c2a47b100841a3f79693120f39dca878125ca315659e118 SHA512 d6cb24afc54d25a96629b9d7445425bf45a5d74f482f1276179753b7e032b0cc57302d152c80c6e054d2ddd2f5b79c74c1b467305636ac9af51cdc7cc8f3fe91 WHIRLPOOL 9f94901c253a30f3b79282b5cfb868d3c298661a9e044526957e8164c71ac751654f5bd2f56c0cfddf71b7b88b3fe57054b2aac5bb2c5a450534bbb6d46d42cf
```

Comment:
I'm sorry. I had no idea about the thin manifest thing.
I think I've got that fixed.